### PR TITLE
Allow to set base server id

### PR DESCRIPTION
**Pull request by gites**
PR for issue https://github.com/oracle/mysql-operator/issues/69

### CODE REVIEW

**Comment by gites** [comment]
PR squashed and added validation of BaseServerID

**Comment by prydie** [comment]
LGTM.

/cc @owainlewis for final review and merge.

File: pkg/apis/mysql/v1/cluster.go [hunk 1]
```diff
@@ -22,8 +22,11 @@ import (
 
 const (
 	// The default MySQL version to use if not specified explicitly by user
-	defaultVersion  = "5.7.20-1.1.2"
-	defaultReplicas = 1
+	defaultVersion      = "5.7.20-1.1.2"
+	defaultReplicas     = 1
+	defaultBaseServerID = 1000
+	// Max safe value for BaseaServerID calculated as max MySQL server_id value - max Replication Group size
```
Comment (owainlewis) [inline pkg/apis/mysql/v1/cluster.go]: Minor typo: BaseaServerID => BaseServerID

**Comment by gites** [comment]
@owainlewis corrected

**Pull request merged**